configs/_base_/datasets/coco_detection.py

```diff
@@ -23,16 +23,16 @@
             dict(type='Resize', keep_ratio=False),
             dict(type='RandomFlip'),
             dict(type='Normalize', **img_norm_cfg),
-            dict(type='Pad', size_divisor=32),
+            # dict(type='Pad', size_divisor=32),
```

Comment:
why commenting out the padding?

Reply:
Because we aren't keeping aspect ratio so we don't need to pad the image